docs/source/upgrade/index.rst

```diff
@@ -91,27 +91,28 @@ Upgrade the ScyllaDB Manager Server and Client
 
 Confirm that the settings for ``scylla-manager.repo`` are correct.
 
-**On the Manager Server** display the contents of the scylla-manager.repo and confirm the version displayed is the desired version, in this example 3.0.
+**On the Manager Server**, display the contents of ``scylla-manager.repo`` to verify the version you're currently running:
```

Comment:
Please verify this change. I THINK this is the reason why we display scylla-manager.repo, but perhaps I'm wrong. 
The previous instruction is not helpful.

Reply:
If the question is why `scylla-manager.repo` was displayed in unformatted form, then yes.